shaders/texture_mipmap_generation/texture.vert

```diff
@@ -0,0 +1,40 @@
+#version 450
+/* Copyright (c) 2019, Sascha Willems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+layout (location = 0) in vec3 inPos;
+layout (location = 1) in vec3 inNormal;
+layout (location = 2) in vec2 inUV;
+
+layout (set = 0, binding = 0) uniform UBO 
+{
+	mat4 projection;
+	mat4 modelview;
+	float lodBias;
+	int samplerIndex;
+} ubo;
+
+layout (location = 0) out vec2 outUV;
+layout (location = 3) out vec3 outNormal;
+
+void main() 
+{
+	outUV = inUV;
+	outNormal = mat3(inverse(transpose(ubo.modelview))) * inNormal;
```

Comment:
The guys on embedded might not like this. Is there a scale in the model/view matrix that makes this necessary?

Reply:
Nope, that's a left-over. Will remove this.

Reply:
@SaschaWillems - We have 3 approvals so we can merge once this is removed.

Reply:
I have removed the normal calculations, the fragment shader doesn't do any lighting calculations, so it wasn't required after all.